<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>뷔퐁의 바늘</string>
  </property>
  <widget class="QPushButton" name="pushButton_100">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>262</y>
     <width>93</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>100번</string>
   </property>
  </widget>
  <widget class="MplWidget" name="Mplwidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_interval">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_length">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_interval1">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>10</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>줄 사이 간격</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_length1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
    </font>
   </property>
   <property name="text">
    <string>바늘 길이</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_prob1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>108</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;확률&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_exN1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>145</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>실험값 </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_thN1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>179</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;이론값(π)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_error1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>213</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>오차(%)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1000">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>262</y>
     <width>93</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>1000번</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>262</y>
     <width>93</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>10번</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_acnt1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>74</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>사건의 횟수</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_trial1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>전체 시행 횟수</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_trial">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>57</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string> :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_acnt">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>91</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_prob">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>128</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_exN">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>162</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_thN">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>196</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_error">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>230</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_des">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>262</y>
     <width>93</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>설명 보기</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
